test/powershell/Modules/Microsoft.PowerShell.Utility/Unblock-File.Tests.ps1

```diff
@@ -0,0 +1,68 @@
+function Test-UnblockFile {
+    try {
+        Get-Content -Path $testfilepath -Stream Zone.Identifier -ErrorAction Stop | Out-Null
+    }
+    catch {
+        if ($_.FullyQualifiedErrorId -eq "GetContentReaderFileNotFoundError,Microsoft.PowerShell.Commands.GetContentCommand") {
+            return $true
+        }
+    }
+    
+    return $false
+}
+
+Describe "Unblock-File" -Tags "CI" {
+
+    BeforeAll {
+        if ( ! $IsWindows )
+        {
+            $origDefaults = $PSDefaultParameterValues.Clone()
+            $PSDefaultParameterValues['it:skip'] = $true
+
+        } else {
+            $testfilepath = Join-Path -Path $TestDrive -ChildPath testunblockfile.ttt
+        }
+    }
+
+    AfterAll {
+        if ( ! $IsWindows ){
+            $global:PSDefaultParameterValues = $origDefaults
+        }
+    }
+
+    BeforeEach {
+        if ( $IsWindows ){
+            Set-Content -Value "[ZoneTransfer]`r`nZoneId=4" -Path $testfilepath -Stream Zone.Identifier
+        }
+    }
+
+    It "With '-Path': no file exist" {
+        try {
+            Unblock-File -Path nofileexist.ttt -ErrorAction Stop
+            throw "No Exception!"
+        } 
+        catch {
+            $_.FullyQualifiedErrorId | Should Be "FileNotFound,Microsoft.PowerShell.Commands.UnblockFileCommand"
+        }
+    }
+
+    It "With -LiteralPath': no file exist" {
+        try {
+            Unblock-File -LiteralPath nofileexist.ttt -ErrorAction Stop
+            throw "No Exception!"
```

Comment:
I have two "cons" (for all comments).
I believe we should follow the [documentation](https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/WritingPesterTests.md#writing-pester-tests) (`throw "No Exception!"
`) and preserve consistent in all of tests that currently use this docs template. If a template is bad, then we should change it and bring all the tests to the new template.

If  `Unblock-File` test returns `FileNotFound`, then this will not help our understanding of the problem's root and why the exception throw. (We should mention a `Zone.Identify` stream and give a link to a MSDN article in the message.) I believe this is redundant, since in any case we will always start with the analysis of the test code and do more manual tests. So I prefer to see `Expected : {True} But was : {False}.` that simple means that a file is not unlocked.